pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -141,46 +151,54 @@ Use the following instructions to install Spark Operator from Kommander Catalog
   ```bash
   kubectl kudo plan status --instance kaptain -n kubeflow
   ```
+### Enable Kaptain with a custom configuration
+
+<!-- Is this an alternative or an additional step to the section above: Create a Git repository for Kaptain? -->
+
+1.  Provide the name of a `ConfigMap` in the `AppDeployment`, which provides custom configuration on top of the default configuration:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps.kommander.d2iq.io/v1alpha2
+    kind: AppDeployment
+    metadata:
+      name: kaptain
+    spec:
+      appRef:
+        kind: App
+        name: kaptain-2.0.0
+      configOverrides:
+        name: kaptain-overrides
+    EOF
+    ```
 
-## Log in to Kaptain
-
-Once all components have been deployed, you can log in to Kaptain:
-
-- Discover the cluster endpoint and copy it to the clipboard.
-  If you are running Kaptain _on-premises_:
-
-  ```bash
-  kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].ip}"
-  ```
-
-  Or if you are running Kaptain on _AWS_:
-
-  ```bash
-  kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].hostname}"
-  ```
-
-- Get the login credentials from Konvoy to authenticate:
+1.  Create the `ConfigMap` with the name provided in the step above, with the custom configuration:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    data:
+      values.yaml: 
+    kind: ConfigMap
+    metadata:
+      name: kaptain-overrides
+    EOF
+    ```
 
-  - For Konvoy 1.x:
+Kommander waits for the `ConfigMap` to be present before deploying the `AppDeployment` to the attached clusters.
 
-    ```bash
-    konvoy get ops-portal
-    ```
+### Add Kaptain 
 
-  - For DKP 2.x:
+Apply the modifications to the yaml file and make Kaptain available in the DKP Catalog apps section:
 
-    ```bash
-    dkp open dashboard
-    ```
+```bash
+kubectl apply -f gitrepository.yaml
```

Comment:
????  This seems out of place here?  Doesn't this go with the ## Add Kaptain to your DKP Catalog Applications via CLI section?

Reply:
@cneth should this go after ### Create a Git repository for Kaptain and before  ### Enable Kaptain with a custom configuration ? Just need a confirmation that I understood this correctly and I will make the changes.

Reply:
not sure we need this whole section, I _think_  above docs cover everything necessary. not sure what "Add Kaptain" is in this case. @lukeogg ?